diff --git a/cmake/FetchDXVK.cmake b/cmake/FetchDXVK.cmake
index f84de82..050b05b 100644
--- a/cmake/FetchDXVK.cmake
+++ b/cmake/FetchDXVK.cmake
@@ -9,12 +9,13 @@ else()
 endif()
 
 ExternalProject_Add(dxvk_git
-    GIT_REPOSITORY "https://github.com/doitsujin/dxvk"
-    GIT_TAG "v2.6.2"
+    URL file://@dxvk-git@
     UPDATE_DISCONNECTED true
     USES_TERMINAL_CONFIGURE true
     USES_TERMINAL_BUILD true
     BUILD_IN_SOURCE true
+    PATCH_COMMAND
+        sed -i -e "s|^#!.*python3|#!@python3@/bin/python3|" <SOURCE_DIR>/subprojects/libdisplay-info/tool/gen-search-table.py
     CONFIGURE_COMMAND
         sed -i s/private://g
             src/dxvk/dxvk_shader.h &&
@@ -51,10 +52,13 @@ target_link_directories(dxvk
     INTERFACE ${SOURCE_DIR}/build/src/dxgi
     INTERFACE ${SOURCE_DIR}/build/src/d3d11
     INTERFACE ${SOURCE_DIR}/build/src/spirv
-    INTERFACE ${SOURCE_DIR}/build/src/util)
+    INTERFACE ${SOURCE_DIR}/build/src/util
+    INTERFACE ${SOURCE_DIR}/build/src/wsi
+    INTERFACE ${SOURCE_DIR}/build/src/vulkan
+    INTERFACE ${SOURCE_DIR}/build/subprojects/libdisplay-info)
 target_include_directories(dxvk SYSTEM
     INTERFACE ${SOURCE_DIR}/build-native/include/dxvk
     INTERFACE ${SOURCE_DIR}/src
     INTERFACE ${SOURCE_DIR}/include/spirv/include)
 target_link_libraries(dxvk INTERFACE
-    -Wl,--start-group dxgi d3d11 util ldxvk dxbc spirv -Wl,--end-group)
+    -Wl,--start-group dxgi d3d11 util ldxvk dxbc spirv wsi vkcommon display-info -Wl,--end-group)
diff --git a/cmake/FetchPeParse.cmake b/cmake/FetchPeParse.cmake
index df7879b..31a4b87 100644
--- a/cmake/FetchPeParse.cmake
+++ b/cmake/FetchPeParse.cmake
@@ -1,8 +1,7 @@
 include(ExternalProject)
 
 ExternalProject_Add(peparse_git
-    GIT_REPOSITORY "https://github.com/trailofbits/pe-parse"
-    GIT_TAG "v2.1.1"
+    URL file://@peparse-git@
     UPDATE_DISCONNECTED true
     USES_TERMINAL_CONFIGURE true
     USES_TERMINAL_BUILD true
